core/trino-main/src/main/java/io/trino/server/security/jwt/JwtAuthenticator.java

```diff
@@ -49,9 +53,12 @@ public JwtAuthenticator(JwtAuthenticatorConfig config, SigningKeyResolver signin
     }
 
     @Override
-    protected Jws<Claims> parseClaimsJws(String jws)
+    protected Optional<Principal> extractPrincipalFromToken(String token)
     {
-        return jwtParser.parseClaimsJws(jws);
+        return Optional.ofNullable(jwtParser.parseClaimsJws(token)
```

Comment:
I think that moving `jwtParser..` part to the next line would create an additional indent and thus improve readability slightly.